assets/xml/objects/object_mm.xml

```diff
@@ -1,68 +1,80 @@
-        <Limb Name="object_mm_Standardlimb_009658" Type="Standard" EnumName="OBJECT_MM_LIMB_09" Offset="0x9658" />
-        <Limb Name="object_mm_Standardlimb_009664" Type="Standard" EnumName="OBJECT_MM_LIMB_0A" Offset="0x9664" />
-        <Limb Name="object_mm_Standardlimb_009670" Type="Standard" EnumName="OBJECT_MM_LIMB_0B" Offset="0x9670" />
-        <Limb Name="object_mm_Standardlimb_00967C" Type="Standard" EnumName="OBJECT_MM_LIMB_0C" Offset="0x967C" />
-        <Limb Name="object_mm_Standardlimb_009688" Type="Standard" EnumName="OBJECT_MM_LIMB_0D" Offset="0x9688" />
-        <Limb Name="object_mm_Standardlimb_009694" Type="Standard" EnumName="OBJECT_MM_LIMB_0E" Offset="0x9694" />
-        <Limb Name="object_mm_Standardlimb_0096A0" Type="Standard" EnumName="OBJECT_MM_LIMB_0F" Offset="0x96A0" />
-        <Skeleton Name="object_mm_Skel_0096E8" Type="Flex" LimbType="Standard" LimbNone="OBJECT_MM_LIMB_NONE" LimbMax="OBJECT_MM_LIMB_MAX" EnumName="ObjectMmLimb" Offset="0x96E8" />
-        <Animation Name="object_mm_Anim_0099B4" Offset="0x99B4" /> <!-- Original name is "mm_runwait" -->
-        <Animation Name="object_mm_Anim_00A4E0" Offset="0xA4E0" /> <!-- Original name is "mm_tatsu" -->
-        <Animation Name="object_mm_Anim_00A8D8" Offset="0xA8D8" /> <!-- Original name is "mm_uzukumaru" -->
-        <Animation Name="object_mm_Anim_00B09C" Offset="0xB09C" /> <!-- Original name is "mm_wait01" -->
-        <Animation Name="object_mm_Anim_00BA78" Offset="0xBA78" /> <!-- Original name is "mm_watasu" -->
-        <Animation Name="object_mm_Anim_00C32C" Offset="0xC32C" /> <!-- Original name is "mm_watasuwait" -->
-        <Animation Name="object_mm_Anim_00C640" Offset="0xC640" /> <!-- Original name is "mm_yoru" -->
-        <Animation Name="object_mm_Anim_00CD90" Offset="0xCD90" /> <!-- Original name is "uttekure" -->
-        <Animation Name="object_mm_Anim_00DA50" Offset="0xDA50" /> <!-- Original name is "yorokobi_long" -->
+
+        <!-- Postman Animations -->
+        <Animation Name="gPostmanSkippingAnim" Offset="0x468" /> <!-- Original name is "hasiruu" -->
+        <Animation Name="gPostmanHeadInHandsAnim" Offset="0x99C" /> <!-- Original name is "mm_kunou" -->
+        <Animation Name="gPostmanSleepingAnim" Offset="0xFC4" /> <!-- Original name is "mm_neru" -->
+        <Animation Name="gPostmanCheckingMailAnim" Offset="0x1F84" /> <!-- Original name is "mm_postcheck" -->
+        <Animation Name="gPostmanRunningAnim" Offset="0x2238" /> <!-- Original name is "mm_run01" -->
+        <Animation Name="gPostmanJoggingInPlaceAnim" Offset="0x99B4" /> <!-- Original name is "mm_runwait" -->
+        <Animation Name="gPostmanStandingUpAnim" Offset="0xA4E0" /> <!-- Original name is "mm_tatsu" -->
+        <Animation Name="gPostmanOnHandsAndKneesAnim" Offset="0xA8D8" /> <!-- Original name is "mm_uzukumaru" -->
+        <Animation Name="gPostmanIdleAnim" Offset="0xB09C" /> <!-- Original name is "mm_wait01" -->
```

Comment:
Opinion: this actor could arguably have different idle animations depending on if hes sitting, laying or standing. This animation is idle standing, and could be named or commented to identify that, but not requiring this to be changed.

Reply:
That makes sense to me, the only thing is if they were to be named idle if including the context would make it a bit more readable. (i.e gPostmanSittingIdle vs gPostmanTimingMinigameIdle)

Reply:
I don't think idle is necessary at all. I try to name animations on what they are doing first and how they are used second, and just renaming to "Standing" would be how I approach this ambiguity. There are multiple actors in the current code base with Idle animations tho, this could go either way.

(also I should point out that while I prefer using ongoing tense "Sitting", the project is trying to use present tense "Sit" for animations, and you will likely be asked to adjust these animations to match)